addons/website/static/src/xml/website.cookies_warning.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<templates xml:space="preserve">
+    <t t-name="website.cookiesWarning">
+        <button class="o_no_optional_cookie w-100 h-100 alert alert-info text-center pt16 pb16 rounded"
+                t-on-click="showCookiesBar">
+            To see the interactive content, you need to accept optional cookies. Click here to accept cookies.
+        </button>
```

Comment:
This `on-click=showCookiesBar` is not straightforward:
- It's a method passed from the caller (the caller being an util method) to dispatch a new Event(showCookiesBar)
- Which is then listened by the cookie bar widget when (non optional) cookies are accepted to then call a method
- And this method will show the cookies bar

Seems like something could be simplified, I will wait for those comments to be processed to see a bit more clearly what's left / what has changed.

But the cookies bar is a regular bootstrap modal, if the public root was in charge of adding those warning in DOM and adding a listener on click, that listener could be about doing `.modal('show');` directly on the cookies bar no? Without the need of an Event and event listeners.
Just thinking out loud at this point